grpc.go

```diff
@@ -752,7 +807,10 @@ func (a *agentGRPC) UpdateRoutes(ctx context.Context, req *pb.UpdateRoutesReques
 }
 
 func (a *agentGRPC) OnlineCPUMem(ctx context.Context, req *pb.OnlineCPUMemRequest) (*gpb.Empty, error) {
-	go onlineCPUMem()
+	if !req.Wait {
```

Comment:
In order to have a fast boot time, the go routine should run inside the VM, otherwise the runtime fails with next error

```
rpc error: code = Unavailable desc = transport is closing.
```

Reply:
Ok makes sense!